lib/rubocop/cop/lint/empty_block.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for blocks without a body.
```

Comment:
I'd add here also some explanation that such empty blocks are typically an oversight, so it's clearer what we're aiming for.